scene/resources/skeleton_profile.cpp

```diff
@@ -91,9 +91,7 @@ bool SkeletonProfile::_get(const StringName &p_path, Variant &r_ret) const {
 		} else {
 			return false;
 		}
-	}
-
-	if (path.begins_with("bones/")) {
+	} else if (path.begins_with("bones/")) {
```

Comment:
I think it might be a bit easier to parse and less error prone to have each top-level `if` scope end with `return true` when it has done some successful handling of the property, instead of just having `return false` and then hoping that through trickling down the chain of `else if`s we won't fail to land on the final return.